src/jit/codegen.h

```diff
@@ -32,7 +32,11 @@ class CodeGen : public CodeGenInterface
     // This could use further abstraction
     CodeGen(Compiler* theCompiler);
 
+#if defined(FEATURE_JIT_DROPPING)
+    virtual void genGenerateCode(void** codePtr, ULONG* totalNCSize, ULONG* nativeSizeOfCode);
```

Comment:
I wonder if the arguments *totalNCSize could be eliminated at least in the jit code in favor of just using the global
totalNCSize from compiler.cpp/compiler.h